src/main/java/uk/gov/ons/census/action/model/dto/FulfilmentRequestDTO.java

```diff
@@ -1,16 +1,17 @@
 package uk.gov.ons.census.action.model.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import java.util.UUID;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data
 @NoArgsConstructor
 public class FulfilmentRequestDTO {
-
   private UUID caseId;
-
   private String fulfilmentCode;
-
   private Contact contact;
+
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  private String individualCaseId;
```

Comment:
why is this `String` when `caseId` is `UUID`?

Reply:
it's more fun to make it a String, then you can write UUID.fromString(...  in the code,  and build the object with UUID.random().toString().   More typing, more fun. WET